### PR TITLE
feature/30 Add configuration for VPC

### LINKED ISSUE #21: Set retry count for scheduled event to 0
The events must not retry, since this could lead to duplicate reporting

### LINKED ISSUE #30: Add configuration for VPC
Currently, users have to reconfigure the VPC of the lambda after its creation.

It would be more convenient to directly have a property for the VPC.

### PR DESCRIPTION
**Pull request by jakobbraun**
Closes #30

### CODE REVIEW

File: doc/changes/changes_0.1.0.md [hunk 1]
```diff
@@ -8,6 +8,7 @@
 * #3: Added SAM deployment
 * #4: Use AWS Secrets Manager for storing the Exasol credentials
 * #7: Added additional metrics: `NET`, `SWAP`, `DB_RAM_SIZE`, `NODES`
+* #30: Add configuration for VPC
```
Comment (morazow) [inline doc/changes/changes_0.1.0.md:11]:
```suggestion
* #30: Added configuration for VPC
```

File: doc/changes/changes_0.1.0.md [hunk 2]
```diff
@@ -24,3 +25,4 @@
 * #8: Fixed wrong metrics
 * #14: Removed failing SAM validate CI, since it requires AWS access
 * #19: Fixed SAM template (`ClassNotFoundException` on deployment)
+* #21: Set retry count for scheduled event to 0
```
Comment (morazow) [inline doc/changes/changes_0.1.0.md:28]:
```suggestion
* #21: Changed retry count for scheduled event to 0
```

**Review by morazow — APPROVED** [review]
Approved with minor suggestions.

**Review by morazow — APPROVED** [review]

**Comment by jakobbraun** [comment]
I just realized that the fix of this bug breaks publishing the application. See https://github.com/aws/serverless-application-model/issues/1931

**Review by morazow — APPROVED** [review]

**Pull request merged**